prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -90,6 +96,12 @@ def before_run
     donate
   end
 
+  label def sync_kubernetes_services
+    decr_sync_kubernetes_services
+    kubernetes_cluster.client.sync_kubernetes_services
```

Comment:
can we put a mechanism here (timeout or number of operations) to limit the execution time so that we avoid possible apoptosis. It's OK to let it go for now but let's put a TODO at least.